.github/workflows/kickstart-tests.yml

```diff
@@ -339,3 +339,26 @@ jobs:
           target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  waive:
+    runs-on: ubuntu-latest
+    needs: pr-info
+    if: needs.pr-info.outputs.allowed_user == 'true' && contains(github.event.comment.body, '--waive')
+    steps:
+
+      - name: Get the waiving reason
+        id: get_reason
+        run: |
+          REASON=$(echo "${{github.event.comment.body}}" | sed -e "s#/status-test --waive ##")
+          echo "::set-output name=reason::Waived, $REASON"
```

Comment:
You shouldn't do this. See line 55:

```
        # Do not use comment body directly in the shell command to avoid possible code injection.
        env:	        env:
          BODY: ${{ github.event.comment.body }}	          BODY: ${{ github.event.comment.body }}
```

Reply:
Thanks for the catch!